inspec.yml

```diff
@@ -60,4 +60,28 @@ inputs:
     type: Array
     required: true
     value:
-      - 'LISTENER'
\ No newline at end of file
+      - 'LISTENER'
+
+# oracle19c-2.2.10
+  - name: sec_max_failed_login_attempts
+    description: 'Number of maximum failed login attempts, in seconds'
+    type: Numeric
+    value: 3
```

Comment:
2.2.10 does not need to be parameterized anymore since there's no directly associated value in ARS